### PR TITLE
Website implementation

### PR DESCRIPTION
**Pull request by AntonioMateoGomez**
This PR sets the structure to have a website for the Bow-arch library, based on the way Bow OpenAPI gets its own one.

These changes add an initial structure and a proper adaptation to its visual identity.

Could you please have a look at what additional changes would need to be made to have the website being deployed?

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,192 @@
+Copyright 2018 The Bow Authors
```
Comment (truizlop) [inline LICENSE]:
```suggestion
Copyright 2020 The Bow Authors
```

File: docs/_config.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#-------------------------
+name: Bow-arch
+#-------------------------
+title: bow-arch # To be used on meta tags mainly
+#-------------------------
+description: bow-arch
+#-------------------------
+author: 47 Degrees
+keywords: functional-programming, monads, monad-transformers, functional-data-structure, swift, bow, xcode, xcode-playgrounds, playgrounds, fp-types, adt, free-monads, tagless-final, mtl, for-comprehension, category-theory
```
Comment (truizlop) [inline docs/_config.yml]:
```suggestion
keywords: functional-programming, monads, monad-transformers, functional-data-structure, swift, bow, xcode, xcode-playgrounds, playgrounds, fp-types, adt, free-monads, tagless-final, mtl, for-comprehension, category-theory, architecture, mobile, apps, application, ios, macos
```

File: LICENSE (hunk 1 above)
Comment (miguelangel-dev) [inline LICENSE]: For now, I think we can remove this file - unless we should update authors...years...

File: docs/index.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+---
+layout: home
```
Comment (miguelangel-dev) [inline docs/index.md:2]: do we need this file?

**Review by miguelangel-dev — APPROVED** [review]
Great work @AntonioMateoGomez. It looks good for me, but I have not any idea about `css` or `ruby`- @calvellido or @israelperezglez could you do a quick review, please?

File: docs/_data/_doc.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    >
+      <img
+        src="{{ 'img/sidebar-icon-open.svg' | relative_url }}"
+        alt="Toggle"
+        title="Toggle"
+      />
+    </button>
+
+    <div class="link-container">
+      <!-- {% for item in site.data.versions %}
```
Comment (calvellido) [inline docs/_data/_doc.html]: Could we remove this?

**Review by calvellido — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 3 above)
Comment (AntonioMateoGomez) [reply docs/index.md:2]: Yes, that´s needed for the home layout

File: docs/_data/_doc.html (hunk 4 above)
Comment (AntonioMateoGomez) [reply docs/_data/_doc.html]: Yes, of course. I realized that file is not necessary

**Comment by israelperezglez** [comment]
@AntonioMateoGomez I have some details: 

**h2**
- {font size} `Review`

**Header button** 
 - {font size} `Review`
- {border-radius} `Review`

**site-footer**
- {border-top}  `same color doc line`

**Review by calvellido — APPROVED** [review]

**Comment by AntonioMateoGomez** [comment]
Ok, Thanks @israelperezglez your review is solved in the last commit

**Pull request merged**